hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java

```diff
@@ -942,6 +942,15 @@
     public static final HazelcastProperty TASK_SCHEDULER_REMOVE_ON_CANCEL =
             new HazelcastProperty("hazelcast.executionservice.taskscheduler.remove.oncancel", false);
 
+    /**
+     * Client operations in a cluster are intercepted on the member side for permission control.
+     * Some API might not hold permission mappings, i.e., inherited JDK methods that are not yet supported, but still available
+     * to the end-user to run, in which case we used to allow execution. This property controls this behaviour, and by default
+     * (after 3.10) will reject any call to API that has no mapped permission.
+     */
+    public static final HazelcastProperty DENY_ACCESS_TO_MISSING_PERMISSION_MAPPING =
+            new HazelcastProperty("hazelcast.deny.access.to.missing.permission.mapping", false);
```

Comment:
I think the doc, the prop name, and its default value are in contradiction. Either the default value should be `true` or the property name should be `hazelcast.allow.access.to.missing.permission.mapping`. I'd prefer the first.